content/creator/wearables-and-emotes/manage-collections/uploading-smart-wearables.md

```diff
@@ -0,0 +1,165 @@
+---
+date: 2023-07-27
+title: Uploading Smart Wearables
+description: Guidelines to Upload Smart Wearables to the Editor
+categories:
+  - Decentraland
+type: Document
+aliases:
+url: /creator/wearables-and-emotes/manage-collections/uploading-smart-wearables
+weight: 4
+---
+
+After generating your smart wearable using the [SDK7](/creator/development-guide/sdk7/smart-wearables/), the next step is to upload it to the builder. This document explains the process of uploading, publishing and putting Smart Wearables up for sale.
+
+#### Uploading Your File
+
+Remember that you need to create a collection before you can upload your file. If you don’t know how to do that, check [Creating a Collection](/creator/wearables-and-emotes/manage-collections/creating-a-collection.md). To upload your smart wearable, just drag and drop the file on the **_New Item_** window or browse your computer. It will automatically detect if the file is a smart wearable. **Remember that the collection max file size is 2MB**.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/01_new_item.png" width="400" />
+
+When you upload the file, you will be asked to upload a video showcase.
+
+#### Uploading Your Video Showcase
+
+This video showcase will be displayed in the marketplace for buyers to see how the Smart Wearable works.
+
+To upload your video showcase, just drag and drop a **.mp4** video file on the **_Upload a video for your smart wearable_** window or browse your computer. **Remember that the video max file size is 4MB and max duration is 15 seconds**.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/01_new_video.png" width="400" />
+
+When you upload the video showcase, you will be asked to enter a name and define the rarity and the category. You can also add the thumbnail for the smart wearable.
+
+### **Rarity**
+
+Select the Rarity of your item.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/03_rarity.png" width="400"/>
+
+| Rarity    | Number of Items |
+| --------- | --------------- |
+| Unique    | 1               |
+| Mythic    | 10              |
+| Legendary | 100             |
+| Epic      | 1,000           |
+| Rare      | 5,000           |
+| Uncommon  | 10,000          |
+| Common    | 100,000         |
+
+### **Category**
+
+Wearables are organized into different categories, depending on what part of an avatar they modify. Select the appropriate category for your item.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/04_category.png" width="400"/>
```

Comment:
WDYT about adding a table like the Rarity section showing the available categories?

Reply:
```suggestion
Wearables are organized into different categories, depending on what part of an avatar they modify. In this step, you must select the appropriate item category.
```